twisted/conch/ssh/keys.py

```diff
@@ -824,6 +878,12 @@ def type(self):
                 'unknown type of object: %r' % (self._keyObject,))
 
 
+    def getECKeyName(self):
```

Comment:
This will always return `None`, since `ecKeyName` is not referenced anywhere else in this pull request.